.github/workflows/ci-e2e-vrt.yml

```diff
@@ -29,14 +29,17 @@ jobs:
             - name: Install package.json dependencies with pnpm
               run: pnpm install --frozen-lockfile
 
+            - name: Install CI utilities with pnpm
+              run: pnpm install concurrently http-server wait-on
```

Comment:
Should be a bit faster than `dlx`, also less flaky (I previously saw an error in CI where `pnpm dlx http-server` was still downloading the package when `pnpm dlx wait-on` timed out.

Reply:
Good idea!